packages/fast-form-generator-react/src/form/form.utilities.ts

```diff
@@ -425,8 +275,22 @@ export function isRootLocation(location: string): boolean {
     return location === "";
 }
 
-export function getComponentTrackerByLocation(props: IFormProps, rootLocation: IComponentItem): IComponentItem[] {
-    return typeof props.location !== "undefined" // Location has been passed
-        ? getComponentTracker(props.location.schemaLocation, props.location.dataLocation, props.schema, [rootLocation])
-        : [rootLocation];
+/**
+ * Gets breadcrumbs from navigation items
+ */
+export function getBreadcrumbs(
+    navigation: INavigationItem[],
+    handleClick: HandleBreadcrumbClick
+): IBreadcrumbItem[] {
+    const breadcrumbs: IBreadcrumbItem[] = [];
```

Comment:
More of a nit, but this is a prefect case for using Array.map - you avoid creating a new variable completely and will read more cleanly